agent.go

```diff
@@ -39,6 +43,10 @@ var agent struct {
 	topic     string      // The default log topic
 }
 
+func init() {
```

Comment:
leftover? Or is this called somewhere?

Reply:
The package needs a logger when running tests. If we don't do this or something similar, then users will need to set up the logger in their tests. Which comes with new problems: when and were to do and we would need to document it.